src/Ng/packages.config

```diff
@@ -11,7 +11,7 @@
   <package id="Microsoft.Web.Xdt" version="2.1.1" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.1.0" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
-  <package id="NuGet.Core" version="2.8.5" targetFramework="net45" />
+  <package id="NuGet.Core" version="2.10.1" targetFramework="net45" />
   <package id="nuget.versioning" version="3.2.0" targetFramework="net45" />
   <package id="SharpZipLib" version="0.86.0" targetFramework="net45" />
```

Comment:
Consider moving the project off packages.config to project.json (still with csproj) that should work if the project doesn't use any content